packages/loopring_v3/contracts/impl/DelayedTransaction.sol

```diff
@@ -102,14 +105,6 @@ abstract contract DelayedTransaction is IDelayedTransaction, ReentrancyGuard
             transaction.value,
             transaction.data
         );
-
-        // Return the same data the original transaction would
-        // (this will return the data even though this function doesn't have a return value in solidity)
-        assembly {
-            switch success
-            case 0 { revert(add(returnData, 32), mload(returnData)) }
-            default { return(add(returnData, 32), mload(returnData)) }
```

Comment:
@Brecht, it seems we also have similiar code in Hebao's BaseWallet.sol:

```solidity
    /// @dev This default function can receive Ether or perform queries to modules
    ///      using bound methods.
    fallback()
        external
        payable
    {
        if (msg.data.length == 0 || msg.value > 0) {
            // Note: don't do anything here so send/transfer still works
            // -- the 2300 gas limit for send/transfer may case out-of-gas
            // issue after Istanbul.
            return;
        }

        address module = methodToModule[msg.sig];
        require(isAddressInSet(MODULE, module), "MODULE_UNAUTHORIZED");

        (bool success, bytes memory returnData) = module.call(msg.data);
        assembly {
            switch success
            case 0 { revert(add(returnData, 32), mload(returnData)) }
            default { return(add(returnData, 32), mload(returnData)) }
        }
    }
```

BTW, what's the difference between the old and the new implementation? It seems to me the old code will return some data in case of success, while the new code does not.